neqo-transport/src/send_stream.rs

```diff
@@ -895,6 +908,70 @@ mod tests {
         assert_eq!(res, (15, None));
     }
 
+    #[test]
+    fn tx_buffer_next_bytes_1() {
+        let mut txb = TxBuffer::new();
+
+        assert_eq!(txb.avail(), 65535);
```

Comment:
A constant for this 65535 is probably a good idea.

Reply:
Maybe at some point. Depends if we ever change TxBuffer::BUFFER_SIZE. Currently the entire test is written with the assumption it is a certain size. That assert catches if it ever changes, at which point really the whole test needs to be altered accordingly. Writing all the steps now on the assumption buffer size will change is not yet worth doing.